### ISSUE TITLE
Improve accessibility score

### ISSUE THREAD

**Issue by arnaud-deprez** [arnaud-deprez/arnaud-deprez.github.io#23]
# Context

Current accessibility is very high but not 100%. That's mainly because our links does not have a sufficient contrast with white background.

# Technical

One quick way without changing style of the whole site is to put link colour a bit darker than the primary colour.
- [ ] Accessibility score should be 100%
